pandunia-lekse.csv

```diff
@@ -1082,40 +1077,50 @@ hene|||zho:(hen), fra:(haine)|hate||odio|||||||||||||malamo|viha|nienawidzić
-hibride||||hybrid (mongrel)||híbrido (mestizo)|||||||||||||miksulo|risteytys (hybridi)|hybryda; kundel
+hibride|||eng:hybrid, spa:por:híbrido, fra:hybride, rus:гибрид (gibrid)|hybrid (mongrel)||híbrido (mestizo)|||||||||||||miksulo|risteytys (hybridi)|hybryda; kundel
 hidre|mate|H||hydrogen||hidrógeno|||||||||||||hidrogeno|vety|wodór
 hijabe|||ara:(ḥijāb), fas:(hejâb), swa:hijabu, hau:hijabi, hin:हिजाब (hijāb), ben:হিজাব (hijab), eng:por:hijab, fra:hidjab, spa:hiyab, rus:хиджаб (xidžab)|veil (cover, screen)|voile|velo|véu|вуаль|حِجَاب|面纱||||||||hijabu (veli, shela)||verho (huntu, peite)|welon (woalka)
 hijeni||||cleanly (hygienic, sanitary)||||||||||||||||hygieeninen|czysty (higieniczny, sanitarny)
 hijenia|||deu:eng:hygiene, fra:hygiène, spa:por:higiene, rus:гигиена (gigiena), tur:hijyen|hygiene (cleanliness)|hygiène|higiene|higiene|гигиена||||||||||hijyen||hygienia (puhtaus)|higiena
 himalia||||Himalayas||Himalaya|||||||||||||Himalajo|Himalaja|Himalaje
 hime|||hin:हिमपात (himpāt), tha:หิมะ (hima), tel:(himamu)|snow|neige|nieve|neve||||||||||||neĝo|lumi|śnieg
-himorose||||frost||escarcha|||||||||||||frosto|kuura|szron
+himrose||||frost||escarcha|||||||||||||frosto|kuura|szron
 hindiste||||Hindu||hindú|||||||||||||hinduo|hindu|hinduistyczny
 hindistia||||Hinduism||hinduismo|||||||||||||hinduismo|hindulaisuus (hinduismi)|hinduizm
 hisaba|||ara:(hasaba), tur:hesap, orm:hisaaba, swa:hisabu|calculate|calculer|calcular|calcular|вычислять|حَسَبَ|计算|計算する|계산하다||||||||laskea (tehdä laskutoimitus)|kalkulować
 hisabia||||arithmetics||aritmética (cálculos)||||||||||||||laskuoppi (aritmetiikka)|arytmetyka
 histore||||history (annals)||historia (pasado)|||||||||||||historio|historia (aikakirjat)|historia; annały, roczniki
 historer||||historian||historiador|||||||||||||historiisto|historioitsija|historyk
-historia||||history (study of history)||historia (estudio de sucesos del pasado)|||||||||||||historio (studo de historio)|historia (oppiaine)|historia, badanie historii
+historia|||eng:history, spa:swa:historia, por:história, fra:histoire, rus:история (istoriya)|history (study of history)||historia (estudio de sucesos del pasado)|||||||||||||historio (studo de historio)|historia (oppiaine)|historia, badanie historii
 ho ci ming site||||Ho Chi Minh City (Saigon)|||||||||Thành phố Hồ Chí Minh||||||||
-hobe||||hobby||pasatiempo (hobby, afición)|||||||||||||hobio|harrastus|hobby
+hobe|||eng:spa:por:deu:fra:hobby, rus:хобби (hobbi)|hobby||pasatiempo (hobby, afición)|||||||||||||hobio|harrastus|hobby
+hoga||||burn (cause to burn)|brûler|quemar|queimar|жечь (палить)|أَحْرَقَ|燃烧|燃やす|태우다||जलाना|||kuchoma|yakmak||polttaa|spalić, palić
+hoge|||spa:fuego, por:fogo + rus:огонь (ogon’), hin:आग (āg), ben:অগ্নি (ôgni), pan:ਅੱਗ (ag) + zho:火 (huǒ), yue:火 (fo2), wuu:火 (hu2), kor:화 (hwa), vie:hoả|fire (burning)|feu|fuego|fogo|огонь|نَار|火|火|불|lửa|आग (अग्नि)|আগুন (অগ্নি)|api|moto|ateş|fajro|tuli|ogień
```

Comment:
I like "hog" very much! It's better than "foge" because it combines all three etymologies perfectly.